Makefile

```diff
@@ -1,4 +1,4 @@
-dev: buffalo adminer migrate
+dev: buffalo redis adminer migrate 
```

Comment:
If you add `redis` to the `depends_on` for the containers that need it (`buffalo` and `test`) you don't need to add anything to `Makefile`. Docker compose does a good job of managing dependencies if they're specified in the `docker-compose.yml` file.

Reply:
Thanks for the tip!